collectors/ebpf.plugin/ebpf_sync.c

```diff
@@ -314,6 +336,13 @@ static void ebpf_create_sync_charts()
  */
 static void ebpf_sync_parse_syscalls()
 {
+    struct section *co = appconfig_get_section(&sync_config, NETDATA_SYNC_CONFIG_NAME);
+    if (co) {
+        struct config_option *cv;
+        for (cv = co->values; cv ; cv = cv->next) {
+        }
+    }
```

Comment:
What for did you add this part?

Reply:
I used this to debug an information, I removed with the last commit. Thank you!